torch/nn/quantized/modules/linear.py

```diff
@@ -136,6 +137,14 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         self.zero_point = int(state_dict[prefix + 'zero_point'])
         state_dict.pop(prefix + 'zero_point')
 
+        version = local_metadata.get('version', None)
+        if version is None or version == 1:
```

Comment:
For my understanding: Do we need to check for version ==1? We dont have this field currently.

Reply:
I think it's better to be explicit, for example in the forward-compatiblity case. It would be an error to enter this branch if someone's loading a newer model, and that would lead to some pretty weird errors

Reply:
Newer models will have version >= 2 right?